# Source: Internal
# Info: Written internally so this component can be built on Solaris.
# Status: This patch is offered at https://sourceforge.net/p/gnu-efi/patches/37/

--- ORIGINAL/gnuefi/reloc_x86_64.c	2016-04-15 14:10:55.424887569 -0700
+++ gnu-efi-3.0/gnuefi/reloc_x86_64.c	2016-04-26 11:33:24.555247600 -0700
@@ -39,6 +39,8 @@
 #include <efilib.h>
 
 #include <elf.h>
+#include <sys/link.h>
+#include <sys/elf_amd64.h>
 
 EFI_STATUS _relocate (long ldbase, Elf64_Dyn *dyn, EFI_HANDLE image, EFI_SYSTEM_TABLE *systab)
 {
